basicmessage_storage/pyproject.toml

```diff
@@ -55,18 +55,17 @@ testpaths = "basicmessage_storage"
 addopts = """
     --quiet --junitxml=./test-reports/junit.xml
     --cov-config .coveragerc --cov=basicmessage_storage --cov-report term --cov-report xml
-    --ruff
```

Comment:
This was throwing errors for me. I actually need to figure out how to do the linting still. Right now i'm just using the ruff vs code extension and not sure if all the rules are being met.